src/index.d.ts

```diff
@@ -46,13 +46,14 @@ export { default as useToggle, type Toggler, type ToggleHandlers } from './hooks
 export { default as useArrowKeyNavigation, type ArrowKeyNavProps } from './hooks/useArrowKeyNavigationHook';
 export { default as useIndexOfLastVisibleChild } from './hooks/useIndexOfLastVisibleChildHook';
 export { default as useIsVisible } from './hooks/useIsVisibleHook';
+export { default as Alert, ALERT_CLOSE_LABEL_TEXT } from './Alert';
+export { default as breakpoints } from './utils/breakpoints';
```

Comment:
Please make the same edits to [`index.js`](https://github.com/openedx/paragon/blob/c60ef2bcc1472b3142036ddd79ceaf299cd67cb6/src/index.js) as well, so the two files stay in sync line for line.